Gaijinizer patch/Map171.txt

```diff
@@ -144,70 +152,77 @@ using magic.#MTLed
 \\C[10]『A-000』と『B-000』\\C[0]は、残念ながら早い時期に
 脱落してしまったよ。
 > CONTEXT: Map171/events/2/pages/0/88/Dialogue/other2
-There were three more candidates in front of him
-......... \\C[10]"A-000" and "B-000"\\C[0] Unfortunately,
-early in time. I've dropped out.#MTLed
+There were three other candidates ahead of him,
+\\C[10]"A-000" and "B-000"\\C[0]. But unfortunately they
+both dropped out of the running quite quickly.#Ron
 > END STRING
 
 > BEGIN STRING
 そして、\\C[10]『C-000』\\C[0]での失敗がいい経験となった。
 
 人間は、悪魔の『性的な誘惑』が一番厄介だと分かったのだ。
 > CONTEXT: Map171/events/2/pages/0/93/Dialogue/other2
-And\\C[10] "C-000"\\C[0] The failure at was a good experience.
-Humans have found that demon's "sexual
-temptation" is the most annoying.#MTLed
+And the failure of \\C[10]"C-000"\\C[0] at least provided
+some useful lessons. We realized that demonic
+temptation really was the biggest hurdle.#Ron
 > END STRING
 
 > BEGIN STRING
 そして、最後の候補者……
 
 最有力候補、\\C[10]『E-000』\\C[0]…………
 > CONTEXT: Map171/events/2/pages/0/97/Dialogue/other2
-And the last candidate...Leading candidate,\\C[10]
-"E-000"\\C[0] ............#MTLed
+Which leads us to the final, leading candidate...
+
+\\C[10]"E-000"\\C[0]...#Ron
 > END STRING
 
 > BEGIN STRING
 それが………
 
 \\N[1]だって言うの！？
 > CONTEXT: Map171/events/2/pages/0/102/Dialogue/other1
-that is………. Say even \\N[1]! ?#MTLed
+No...
+
+Not \\N[1]!!#Ron
 > END STRING
 
 > BEGIN STRING
 彼の目覚めが遅いのも無理はない。
 
 彼の体内にある『魔力』は、特殊なものだからね。
 > CONTEXT: Map171/events/2/pages/0/111/Dialogue/other2
-It is no wonder that his awakening is late.
-"Magic" in his body is a special thing.#MTLed
+It's no wonder he's been so slow to wake.
+
+The magic contained within him is quite special.#Ron
 > END STRING
 
 > BEGIN STRING
 人間が適応し、覚醒するには、少々時間がかかるのだ。
 > CONTEXT: Map171/events/2/pages/0/115/Dialogue/other2
-It takes some time for humans to adapt and wake
-up.#MTLed
+It takes some time for humans to adapt to it and
+wake up.#Ron
 > END STRING
 
 > BEGIN STRING
 ……\\C[10]『天使の魔力』\\C[0]。
 
 それが\\N[1]君の中に眠る、魔力の根元。
 > CONTEXT: Map171/events/2/pages/0/118/Dialogue/other2
-......\\C[10] "Angel's Magic"\\C[0] .
-That sleeps in \\N[1], the root of magic.#MTLed
+Sleeping inside him is the root of all magic...
+
+\\C[10]Angelic Magic\\C[0].#Ron
 > END STRING
 
 > BEGIN STRING
 そう………
 
 君の封印していた魔力だよ。レイナ君。
 > CONTEXT: Map171/events/2/pages/0/122/Dialogue/other2
-so………. It's the magical power you sealed.
-Reina-kun.#MTLed
+That's right...
+
+It's the magic you sealed away inside yourself,
+Reina-kun.#Ron
 > END STRING
```

Comment:
Note

Reply:
I think she's referring to the fact that Reina sealed her own magic away as part of her plan to kill herself since she couldn't die until she'd done that. MOW then took that magic after knocking her unconscious and put it in Kazuya.

The 君の封印していた魔力 could be shortened to 君の魔力 (your magic), with 封印していた (was sealed away) then placed in-between as a modifier to 魔力 (magic power) to get a literal translation of "your sealed away magic power".

Reply:
Ah, that makes sense. I think this wording will be clearer, then:
>_**It's the magic that you had sealed away inside
yourself, Reina-kun.**_